app/src/lib/Map.svelte

```diff
@@ -50,8 +50,27 @@
         return div
     }
 
+    const getUserLocation = () => {
```

Comment:
rename this to something like `initilizeUserTracking`. `getUserLocation` implies it returns something and doesn't have an effect.